src/sortable/sortable.js

```diff
@@ -627,9 +627,9 @@ export default class Sortable extends Draggable {
 			} else if (nodeName === 'img') {
 				element.setAttribute('src', this.currentItem.getAttribute('src'));
 			}
-			style(element, {
-				visibility: 'hidden'
-			});
+			//style(element, {
+			//	visibility: 'hidden'
```

Comment:
Can you elaborate about this one?

Reply:
FYI: today, I did not find time to answer your question - I'll do so by tomorrow. Sorry for the delay...

Reply:
Oh, I just looked into the code and remembered: I had to make the placeholder visible in order to see where the dragged list element could/should be inserted. Unless you have a better alternative, I'd recommend to keep the placeholder visible...